index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Trial Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main id="main-doc">
+        <nav id="navbar">
+            <header>
+                <strong>Coding</strong>
+            </header><br>
+            <div class="nav-link">
+                <a href="#Galvanize_is_the_first_one"> Galvanize is the first one</a>
+            </div>
+            <div class="nav-link">
+                <a href="#Hack_Reactor_is_the_second">Hack Reactor is the second</a>
+            </div>
+            <div class="nav-link">
+                <a href="#Udemy_is_the_third">Udemy is the third</a>
+            </div>
+            <div class="nav-link">
+                <a href="#Codecademy_is_the_fourth_one">Codecademy is the fourth one</a>
+            </div>
+            <div class="nav-link">
+                <a href="#Codecademy_is_the_fourth_one">Codecademy is the fourth one</a>
+            </div>
+
+        </nav><br><br>
+        <div id="technical-doc-page">
+            <section class="main-section">
+                <header>4 best (free) online coding courses for beginners</header>
+                <p>The prospect of learning how to code can be overwhelming, especially if you don’t have any computer
+                    science knowledge or experience.</p>
+
+                <p>But software engineering, data science, and related fields can be stable, long-term career paths no
+                    matter where you live — there’s a huge demand right now for employees with computer science skills
+                    at established companies and startups.</p>
+                <p>Plus, jobs in these fields tend to pay well and come with lots of associated perks and benefits.</p>
+                <p>If you’re mulling over the idea of a new career, learning how to code is a great investment in
+                    yourself.</p>
+            </section>
+
+            <section class="main-section">
+                <header>Galvanize is the first one</header>
```

Comment:
Instead of Header tag for each title, use h1 tag.

Reply:
same here, but also kindly explain why this change is needed

Reply:
Nice job 👍